src/lib/api/Validator.ts

```diff
@@ -75,6 +75,15 @@ class Validator {
     }
   }
 
+  async getFeeRecipientAddress(publicKey: string) {
+    try {
+      const url = `${String(config.links.SSV_API_ENDPOINT)}/accounts/${publicKey}`;
+      return await this.getData(url);
+    } catch (e) {
+      return null;
+    }
+  }
+
```

Comment:
This method should be in a new API class called `Account` -> `lib/api/Account.ts`